packages/mobile/src/transactions/TransactionsList.tsx

```diff
@@ -128,7 +128,7 @@ function mapExchangeStandbyToFeedItem(
     const amountCurrency = amount.currencyCode as Currency
     return feedType === FeedType.EXCHANGE
       ? amountCurrency === Currency.Celo
-      : STABLE_CURRENCIES.includes(amountCurrency)
+      : STABLE_CURRENCIES.some((stableCurrency) => stableCurrency === amountCurrency)
```

Comment:
With the previous thing I was getting this TS error:
```
Argument of type 'Currency' is not assignable to parameter of type 'StableCurrency' .ts(2345)
```
This is because I added the type `StableCurrency[]` to `STABLE_CURRENCIES`. I don't remember exactly why I did that, but it seems ok?